fn/client/src/components/login/Login.js

```diff
@@ -17,13 +23,31 @@ const USER_DATA = {
     password: ''
 };
 
+const SignupSchema = yup.object().shape({
+    email: yup.string()
+        .required("Required")
+        .matches(/^([a-z0-9_-]+\.)*[a-z0-9_-]+@[a-z0-9_-]+(\.[a-z0-9_-]+)*\.[a-z]{2,6}$/, "Email must be correct. Example: nick@mail.com"),

```

Comment:
Move all Regexp to constant
as
const EmailValidationReg = new RegExp(/^([a-z0-9_-]+\.)*[a-z0-9_-]+@[a-z0-9_-]+(\.[a-z0-9_-]+)*\.[a-z]{2,6}$/);